content/en/docs/setup/additional-setup/config-profiles/index.md

```diff
@@ -8,7 +8,7 @@ keywords: [profiles,install,helm]
 ---
 
 This page describes the built-in configuration profiles that can be used when
-[installing Istio using helm](/docs/setup/install/helm/).
+[installing Istio](/docs/setup/install/operator/).
```

Comment:
Should we be calling this operator? Perhaps istioctl would be more appropriate?

Reply:
I'd prefer to leave it alone for now. There will be more refactoring of the setup instructions in followup PRs. I think we can figure out what best to name it then.

Reply:
ok